src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1086,9 +1098,9 @@ public static bool IsDefaultExpressionContext(this SyntaxTree syntaxTree, int po
 
 #if !CODE_STYLE
             if (token.IsKind(SyntaxKind.CommaToken) &&
-                token.Parent.IsKind(SyntaxKindEx.FunctionPointerType, out FunctionPointerTypeSyntax funcPtrType))
+                token.Parent.IsKind(SyntaxKindEx.FunctionPointerParameterList, out FunctionPointerParameterList funcPtrParamList))
```

Comment:
Is this unifiable now with the code above in some way now that the structure is similar?

Reply:
No, `FunctionPointerParameterList` and `ParameterListSyntax` don't share a common base type.

---
In reply to: [451057978](https://github.com/dotnet/roslyn/pull/45684#discussion_r451057978) [](ancestors = 451057978)